assets/js/feature-tours/shared-key-metrics.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * Site Kit by Google, Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/*
+ * Internal dependencies
+ */
+import {
+	VIEW_CONTEXT_MAIN_DASHBOARD,
+	VIEW_CONTEXT_MAIN_DASHBOARD_VIEW_ONLY,
+	VIEW_CONTEXT_ENTITY_DASHBOARD,
+	VIEW_CONTEXT_ENTITY_DASHBOARD_VIEW_ONLY,
+} from '../googlesitekit/constants';
+
+const sharedKeyMetrics = {
+	slug: 'sharedKeyMetrics',
+	contexts: [
+		VIEW_CONTEXT_MAIN_DASHBOARD,
+		VIEW_CONTEXT_MAIN_DASHBOARD_VIEW_ONLY,
+		VIEW_CONTEXT_ENTITY_DASHBOARD,
+		VIEW_CONTEXT_ENTITY_DASHBOARD_VIEW_ONLY,
+	],
+	version: 'n.e.x.t',
```

Comment:
The `version` isn't considered for on-demand tours, so if this stays as such, we can simply remove this line.

For normal tours, this should be the version that the feature it's showing was introduced. This won't be for a while yet so in this case we'd use a future version to make sure it continues to be shown, and then we'd update it to be the version it is released to all users in.